docs/about/whats-new/whats-new-0.7.0.ipynb

```diff
@@ -0,0 +1,466 @@
+   "outputs": [],
+   "source": [
+    "sc.vector(value=[1,2,3])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "vecs = sc.vectors(dims=['x'], unit='m', values=np.arange(12).reshape(4,3))\n",
+    "vecs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "vecs.values"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "vecs.fields.y"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "vecs.fields.z += 0.666 * sc.units.m\n",
+    "vecs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Binned data buffer access\n",
+    "\n",
+    "The internal buffer holding the \"events\" underlying binned data was made previously available as `data.bins.constituents['data']`.\n",
+    "This can now be accessed directly using the new `events` property:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import scipp as sc\n",
+    "import numpy as np\n",
+    "N = int(800)\n",
+    "data = sc.DataArray(\n",
+    "    data=sc.Variable(dims=['event'], values=np.random.rand(N), unit='K'),\n",
+    "    coords={\n",
+    "        'x':sc.Variable(dims=['event'], values=np.random.rand(N)),\n",
+    "        'y':sc.Variable(dims=['event'], values=np.random.rand(N)),\n",
+    "        'z':sc.Variable(dims=['event'], values=np.random.rand(N))\n",
+    "    })\n",
+    "binned = sc.bin(data, edges=[sc.Variable(dims=['x'], values=np.linspace(0.0,1.0,num=5)),\n",
+    "                             sc.Variable(dims=['y'], values=np.linspace(0.0,1.0,num=5)),\n",
+    "                             sc.Variable(dims=['z'], values=np.linspace(0.0,1.0,num=5))])\n",
+    "binned.events"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that there is no guarentee about the order of events in this internal buffer.\n",
+    "Furthermore, it includes all data including potentially reserved zones between bins, or events from bins that are not part of the current variable, for example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x_slice = binned['x', 0] \n",
+    "x_slice.events  # ALL events, including those at different `x`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `events` property should thus be used with care.\n",
+    "\n",
+    "`events` is `None` for data that is not binned:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "binned.bins.sum().events is None"
+   ]
+  }
+ ],
```

Comment:
I can contribute an example for the `redraw` function.

Reply:
Is it useful without streaming?

Reply:
It might be? I don't know, some users might come up with a use case that we didn't think of.
In any case, I think if we are talking about about a new feature in the intro, it makes sense to at least give a small example showing how to use it (at least to give the syntax).

Reply:
We could also do it in the scippneutron "What's New"?

Anyway, maybe we should document it first?

Reply:
> Anyway, maybe we should document it first?

That would probably make more sense. Not sure where I would put that. In a notebook with a mix of different plotting tricks? Is there anything else you can think of that would go in there? Things that people often ask about?
Maybe plotting a slice to select a x or y range?

Reply:
Hmm, but how is that better than recreating the plot?

Maybe we should first gain some more experience with this ourselves (it may require changes, once we use it in practice) first?